air/src/constraints/chiplets/memory/mod.rs

```diff
@@ -174,16 +238,36 @@ fn enforce_values<E: FieldElement>(
 trait EvaluationFrameExt<E: FieldElement> {
     // --- Column accessors -----------------------------------------------------------------------
 
-    /// Gets the value of the specified selector column in the current row.
-    fn selector(&self, idx: usize) -> E;
-    /// Gets the value of the specified selector column in the next row.
-    fn selector_next(&self, idx: usize) -> E;
+    /// The value of the read/write column in the current row.
+    ///
+    /// 0: write, 1: read
+    fn is_read(&self) -> E;
+    /// The value of the read/write column in the next row.
+    ///
+    /// 0: write, 1: read
+    fn is_read_next(&self) -> E;
+    /// The value of the element/word column in the current row.
+    ///
+    /// 0: element, 1: word
+    fn is_word_access(&self) -> E;
+    /// The value of the element/word column in the next row.
+    ///
+    /// 0: element, 1: word
+    fn is_word_access_next(&self) -> E;
     /// The current context value.
     #[allow(dead_code)]
     fn ctx(&self) -> E;
     /// The current address.
     #[allow(dead_code)]
-    fn addr(&self) -> E;
+    fn word_next(&self) -> E;
```

Comment:
should we remove these if they are not used?

Reply:
Agreed, removed